### ISSUE TITLE
Failed to load library error

### ISSUE THREAD

**Issue by rickstaa** [mcevoyandy/tf_visual_tools#6]
When I add the tf_visual_tools panel according to the README.md I get the following error:

```
The class required for this panel, 'tf_visual_tools/TFVisualTools', could not be loaded.
Error:
Failed to load library /home/ricks/panda_autograsp_ws/devel/lib//libtf_visual_tools_gui.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library (Poco exception = /home/ricks/panda_autograsp_ws/devel/lib//libtf_visual_tools_gui.so: undefined symbol: _ZTVN15tf_visual_tools13saveLoadTFTabE
```

Since there is no melodic-devel branch I suspect this has to do with a ROS version incompatibility in the CMakeList.txt file.

#### System information
**OS:** Ubuntu 18.04
**ROS version:** Melodic
**TF_VISUAL_TOOLS version:** Latest

**Comment by rickstaa** [comment]
Closing this issue due to inactivity.

**Comment by artemiialessandrini** [comment]
Having the same issue, is there Melodic support coming soon?

**Comment by rickstaa** [comment]
@artemiialessandrini I was wondering the same. Maybe @mcevoyandy knows more about whether the package is still maintained and open for contributions?

**Comment by mcevoyandy** [comment]
@rickstaa sorry, I'm no longer at PickNik so have not been not paying attention to this repo. Could you try the fix mentioned above? Maybe @mlautman or @nbbrooks could review and merge this for you.

**Comment by mitchallain** [comment]
@mcevoyandy @rickstaa please see my PR #8, I believe it fixes the issue.

**Comment by rickstaa** [comment]
Fix verified on ROS melodic (Ubuntu 18.04). @MitchAllain Thanks a lot for fixing the issue!
